openmc/lib/material.py

```diff
@@ -224,6 +215,32 @@ def add_nuclide(self, name, density):
         """
         _dll.openmc_material_add_nuclide(self._index, name.encode(), density)
 
+    def get_density(self, units='atom/b-cm'):
+        """Get density of a material.
+
+        Parameters
+        ----------
+        units : {'atom/b-cm', 'g/cm3'}
+            Units for density
+
+        Returns
+        -------
+        float
+            Density in requested units
+
+        """
+        if units == 'atom/b-cm':
+            return self.densities.sum()
+        elif units == 'g/cm3':
+            density = c_double()
+            try:
+                _dll.openmc_material_get_density(self._index, density)
+            except OpenMCError:
```

Comment:
I'm curious as to why we want to protect against a raised error here.

Reply:
Hmm, good question. I think that can safely be removed since there is already an error handler on the C API call. I'll go ahead and remove it.